dastardcommander/demo/scrollbar/demo.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
What is the point of this demo being included in dastard commander?

If there is a point, lets put an extensive comment near the top of the file explaining the point.

Reply:
Sorry. This is work-in-progress. I forgot it was going to update the PR. Anyway, point is I could not come anywhere close to adding the much-needed scrollbar to the Observe tab's grid tab. This is a demo for that...

Reply:
ohh, i'll just ignore this pr for a while until you ping me